src/ErrorBoundary.js

```diff
@@ -18,10 +18,12 @@ type State = {
   info: ?ErrorInfo,
 };
 
-class ErrorBoundary extends Component {
+class ErrorBoundary extends Component<Props, State> {
   props: Props;
   state: State;
```

Comment:
These 2 property annotations aren't necessary if you're using the `<Props, State>` annotation form.